usr.sbin/bsdinstall/partedit/part_wizard.c

```diff
@@ -274,9 +274,19 @@ wizard_partition(struct gmesh *mesh, const char *disk)
 			gpart_destroy(gpart);
 		}
 
+		/*
+		 * For CheriBSD/Morello, don't bother prompting for the
+		 * partition type when using the entire disk -- always use
+		 * GPT.  This should possibly be if defined(__aarch64__)
```

Comment:
I’ll simplify the comment, but will otherwise leave the code as is for this release. It will be interesting to see if it ever comes up that anyone wants another partition type with CheriBSD...